src/sentry/static/sentry/app/views/settings/incidentRules/triggers/actionsPanel/index.tsx

```diff
@@ -340,17 +357,36 @@ const ActionsPanelWithSpace = styled(ActionsPanel)`
   margin-top: ${space(4)};
 `;
 
+const PanelItemWrapper = styled(`div`)`
+  padding: ${space(0.5)} ${space(2)} ${space(1)};
+`;
+
 const PanelItemGrid = styled(PanelItem)`
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr 1fr min-content;
+  display: flex;
   align-items: center;
-  grid-gap: ${space(2)};
-  padding: ${space(0.5)} ${space(2)} ${space(1)};
+  gap: ${space(2)};
+  padding: ${space(1)};
   border-bottom: 0;
+  justify-content: between;
+`;
+
+const PanelItemSelects = styled('div')`
+  display: flex;
+  width: 100%;
+  gap: 15px;
```

Comment:
This isn't well supported, ~69% globally - https://caniuse.com/flexbox-gap

Reply:
👍